pom.xml

```diff
@@ -268,15 +265,13 @@
                         <param>hashCode</param>
                         <param>equals</param>
                     </excludedMethods>
-                    <mutationThreshold>55</mutationThreshold>
+                    <excludedClasses>
+                        <param>com.feedzai.openml.util.algorithm.GenericAlgorithm</param>
```

Comment:
why?

Reply:
It's an enum, there's not a lot to test. I can't remember what I saw on the report, but I think it was kinda of useless. Will re-check and post the conclusions here

Reply:
ok, the report says 0% LoC coverage and 0% mutation coverage. I removed this class because it has no logic that can be tested...but I can remove this config if you prefer